css/dashboard.css

```diff
@@ -146,3 +178,161 @@ main .card {
   padding: 1rem;
   margin-right: 0.5rem;
 }
+
+tbody tr:not(:first-child) {
+  border-top: 1px solid #eaedf3;
+}
+
+tr th {
+  font-size: 14px;
+  line-height: 16px;
+
+  /* Black */
+  color: #222222;
+}
+
+tr td {
+  padding: 1.5rem 0.25rem !important;
+  /* padding-bottom: 1rem !important; */
+}
+.bd-example {
+  position: relative;
+  padding: 1rem;
+  margin: 1rem -15px 0;
+  border: solid #f8f9fa;
+  border-width: 0.2rem 0 0;
+}
+
+.result__details .result__detail:not(:first-child) {
+  border-top: 1px solid #eaedf3;
+}
+
+.result__detail {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".result__detail" to come before selector ".result__details .result__detail:not(:first-child)" (no-descending-specificity)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5469161)